canopy/canopyjs/.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "extensions": [".js", ".ts"]
+      }
+    }
+  },
+    "env": {
+      "es6": true,
+      "jest": true,
+      "browser": true
+    },
+    "rules": {
+      "import/no-default-export": 2,
+      "import/prefer-default-export": 0
+    },
+    "extends": ["plugin:@typescript-eslint/recommended", "airbnb", "plugin:prettier/recommended"]
+  }
```

Comment:
It looks like some of the tabs are off in this file and the other JSON files

Reply:
Added the correct extensions to the format script in package.json